### DISCUSSION TITLE
奇忽局 共识1.0

### DISCUSSION THREAD

**Discussion by Lupingyue** [qihuju/qihuju.github.io#12] [Ideas]
1.【定义】奇忽局社区类似一些开源社区的机制，比如GPL的开源机制。奇忽局是使用github/clubhouse这些新一代互联网合作交流工具的华人版e-flux。奇忽局本身更像是一个开源的思想及思想工具的孵化器。奇忽局将发展成线上的去中心化的赛博新车间。

2.【规则】每位合作者本身在做的事情如果跟奇忽局有交叉，则可拿到奇忽局内来讨论、合作、测试、进一步开发开源的部分。

  2.1. 基金会&理事会：

   - 参考维基百科 等成熟案例
   - 设定几个必要的子项目理事会，有时间的人可以自由合理分工
   - 记录内部会议（讨论用于构建，需要有效率）和开发进展所代表的时间价值

  2.2. 资金&资源来源：

   - 捐款、捐赠时间、 dev grant、共赢、 回馈方式
   - 产品的购买价值。 音频-播客、图像-文字-杂志-书、游戏 门票、社区邀请码、nft限量版营销
   - 分红&二级市场 次要激励 以likecoin作为一种实验
   - 以成就感激励

3.【开源】但要保证在奇忽局完成的部分是完全开源的，进而奇忽局别的成员也可以利用这部分积累去发展自己与奇忽局有交叉的事情「[@askender 方案](https://github.com/qihuju/qihuju.github.io/discussions/5#discussioncomment-569552)」

4.【案例】

  4.1. e-flux在九十年代初就只是一群最先使用电子邮件进行交流的策展人与艺术家，其最初的“数字存在”就是一个email list，而它逐渐拥有影响力后才逐渐发展成为涵盖当代艺术各个方面的平台。

  4.2. 另外一个学习对象是新车间——其本身是非盈利组织，但很多的社区成员都是在新车间里与别人合作交流后，集结成新的创业团队出去单独发展，因此新车间在上海创客群体里仍然有着相当大的影响力。

  4.3. [罗伯特议事原则](https://github.com/qihuju/qihuju.github.io/discussions/5#discussioncomment-570489)

  注：以上这组定义来自[MING对柏舟“三条路线”讨论的回应](https://github.com/qihuju/qihuju.github.io/discussions/5#discussioncomment-567082)，我比较认同这个方向。看看其他人有什么不同意见？或者有什么补充？

**Comment by Lupingyue** [comment]
目前的四个条目都是比较粗线条的定义和描述，[柏舟对这四个条目都有一些有价值的反馈](https://github.com/qihuju/qihuju.github.io/discussions/5#discussioncomment-569345)，大家请点击链接进入查看。

刚才跟 @askender 在clubhouse里针对性地讨论了几个问题，我大致罗列一下，更有逻辑和结构化的提炼等柏舟的补充「[关于『资金&资源来源』和『基金会&理事会』](https://github.com/qihuju/qihuju.github.io/discussions/5#discussioncomment-569499)」：
1，社会企业和创意产业的启发，奇忽局的“快”与“慢”的两种建设方式
2，多样性的激励机制
3，多线程的场景
4，奇忽局产品以及产品变现的多种形态
5，成了多个2人or2人以上的奇忽局理事会，提升社区建设效率：奇忽局社区建设理事会，卡牌游戏理事会，wiki理事会，赛博乡村理事会，变现理事会等。自愿报名，自愿退出，以参与时间「会议次数，贡献时间等」来量化贡献

**Reply by askender** [reply]
嗯，我的初步观点写在： https://github.com/qihuju/qihuju.github.io/discussions/5#discussioncomment-569499

奇忽局社区建设理事会， 报名
卡牌游戏理事会， 报名
wiki理事会， 报名
变现理事会，这个命名太。。。 改名为： 「可持续发展理事会」吧？改名后，我报个名

量化贡献这一块表述得有点简陋和不精确（量不如质关键，先有记录就好），之后可以完善一下。。

**Reply by Lupingyue** [reply]
> 嗯，我的初步观点写在： [#5 (comment)](https://github.com/qihuju/qihuju.github.io/discussions/5#discussioncomment-569499)
> 
> 奇忽局社区建设理事会， 报名
> 卡牌游戏理事会， 报名
> wiki理事会， 报名
> 变现理事会，这个命名太。。。 改名为： 「可持续发展理事会」吧？改名后，我报个名
> 
> 量化贡献这一块表述得有点简陋和不精确（量不如质关键，先有记录就好），之后可以完善一下。。

哈哈哈哈哈哈好的呀，已经建了，请加入

**Reply by Tz-H** [reply]
@Lupingyue 

针对Markdown格式的一点说明：如果使用`.`而非`，` 来作为数字标号，可以实现Markdown格式所规范的数字列表，e.g.

```
1，社会企业和创意产业的启发，奇忽局的“快”与“慢”的两种建设方式
2，多样性的激励机制
3，多线程的场景
4，奇忽局产品以及产品变现的多种形态
5，成了多个2人or2人以上的奇忽局理事会，提升社区建设效率：奇忽局社区建设理事会，卡牌游戏理事会，wiki理事会，赛博乡村理事会，变现理事会等。自愿报名，自愿退出，以参与时间「会议次数，贡献时间等」来量化贡献
```

1. 社会企业和创意产业的启发，奇忽局的“快”与“慢”的两种建设方式
2. 多样性的激励机制
3. 多线程的场景
4. 奇忽局产品以及产品变现的多种形态
5. 成了多个2人or2人以上的奇忽局理事会，提升社区建设效率：奇忽局社区建设理事会，卡牌游戏理事会，wiki理事会，赛博乡村理事会，变现理事会等。自愿报名，自愿退出，以参与时间「会议次数，贡献时间等」来量化贡献

**Comment by Lupingyue** [comment]
关于开源协议的选择, [@askender 有了精确的定义](https://github.com/qihuju/qihuju.github.io/discussions/5#discussioncomment-569552)

**Comment by Tz-H** [comment]
简单调整了一下主贴的标点符号
